tdrs-backend/tdpservice/data_files/test/test_serializers.py

```diff
@@ -4,25 +4,41 @@
 
 from tdpservice.data_files.errors import ImmutabilityError
 from tdpservice.data_files.serializers import DataFileSerializer
-from tdpservice.data_files.validators import validate_file_extension
+from tdpservice.data_files.validators import (
+    validate_file_extension,
+    validate_file_infection
+)
 
 
 @pytest.mark.django_db
-def test_serializer_with_valid_data(data_file_data):
+def test_serializer_with_valid_data(data_file_data, user):
     """If a serializer has valid data it will return a valid object."""
-    create_serializer = DataFileSerializer(data=data_file_data)
+    create_serializer = DataFileSerializer(
+        context={'user': user},
```

Comment:
We also need to manually pass this context in tests as it is used in validation steps.